soc/arm/microchip_mec/mec172x/reg/mec172x_regs.h

```diff
@@ -0,0 +1,2105 @@
+/*
+ * Copyright (c) 2009-2018 Arm Limited. All rights reserved.
+ * Copyright (c) 2021 Microchip Technology Inc. and its subsidiaries.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef MEC172X_REGS_H
+#define MEC172X_REGS_H
+
+#include <stddef.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum IRQn {
```

Comment:
do we need this, should irq's be coming from devicetree?

Reply:
Good idea.  I'll remove the IRQ enum.

Reply:
I removed the enum from my header.  Use DT macros to get the NVIC IRQ number.